client/src/bundles/UserForms.js

```diff
@@ -529,6 +529,7 @@ jQuery(document).ready(($) => {
         $(form).removeClass('dirty');
 
         form.submit();
+        userform.$el.trigger('userform.form.submit');
```

Comment:
Would it be more useful before the submit call possibly?

Reply:
The reason I put it there was I wanted a surefire way of knowing that the form was 100% being submitted. I'd be happy to move it above if you want though.

Reply:
No, all good. THanks